packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -59,20 +103,40 @@ QUnit.module('init', {
     });
 
-    QUnit.test('role on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('correct wrapper aria attributes load when showIndicator = false', function(assert) {
+        const label = messageLocalization.format('Loading');
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: false }).dxLoadPanel('instance');
 
-        assert.strictEqual(instance.$wrapper().attr('role'), 'alert');
+        assert.strictEqual($loadPanel.$wrapper().attr('role'), 'alert', 'role attribute has value alert ');
+        assert.strictEqual($loadPanel.$wrapper().attr('aria-label'), label, 'aria-label attribute has value Loading...');
+    });
+
+    QUnit.test('wrapper aria attributes must not load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: true });
+
+        assert.strictEqual($loadPanel.attr('role'), undefined, 'role is not set');
+        assert.strictEqual($loadPanel.attr('aria-label'), undefined, 'aria-label is not set');
+    });
+
+    QUnit.test('correct load indicator aria attributes load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: true }).dxLoadPanel('instance');
+        const label = messageLocalization.format('Loading');
+        const indicatorAttributes = Array.from($loadPanel.$content().find(LOADINDICATOR_CLASS).prop('attributes')).map(attr => attr.name);
+        const expectedAttributes = ['role', 'aria-label'];
+
+        assert.strictEqual($loadPanel.$content().find(LOADINDICATOR_CLASS).attr('role'), 'alert', 'role attribute has value alert');
+        assert.strictEqual($loadPanel.$content().find(LOADINDICATOR_CLASS).attr('aria-label'), label, 'aria-label attribute has value Loading...');
+        assert.strictEqual(expectedAttributes.every(item => indicatorAttributes.includes(item)), true, 'root element has complete set of expected attributes');
     });
 
     QUnit.test('aria-label on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), undefined);
+        assert.strictEqual($loadPanel.$wrapper().attr('aria-label'), undefined, 'aria-label attribute is not set');
 
-        instance.option({ message: '' });
+        $loadPanel.option({ message: '' });
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), 'Loading...');
+        assert.strictEqual($loadPanel.$wrapper().attr('aria-label'), 'Loading...', 'aria-label attribute has value Loading...');
     });
 
     QUnit.test('aria-label on wrapper in Material based themes', function(assert) {
```

Comment:
I think this test can be deleted, because you have removed isMaterialBased from getting aria-label function.

Reply:
i completely agree with you..
[b0fc164](https://github.com/DevExpress/DevExtreme/pull/27080/commits/b0fc16436a7a5e17f74c1a2dfba2b2e3c17154ef)
this is the commit for this one